packages/contracts/contracts/BaseVerifier.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IVerifier} from './interfaces/IVerifier.sol';
+
+contract BaseVerifier {
+    IVerifier verifier;
+
+    uint256 public constant SNARK_SCALAR_FIELD =
+        21888242871839275222246405745257275088548364400416034343698204186575808495617;
+
+    struct EpochKeySignals {
+        uint256 revealNonce;
+        uint256 stateTreeRoot;
+        uint256 epochKey;
+        uint256 data;
+        uint256 nonce;
+        uint256 epoch;
+        uint256 attesterId;
+    }
+
+    struct ReputationSignals {
+        uint256 stateTreeRoot;
+        uint256 epochKey;
+        uint256 graffitiPreImage;
+        uint256 proveGraffiti;
+        uint256 nonce;
+        uint256 epoch;
+        uint256 attesterId;
+        uint256 revealNonce;
+        uint256 proveMinRep;
+        uint256 proveMaxRep;
+        uint256 proveZeroRep;
+        uint256 minRep;
+        uint256 maxRep;
+    }
+
+    error UserAlreadySignedUp(uint256 identityCommitment);
```

Comment:
Some of these errors are not used in these contracts. Try to only include errors that are needed.

Reply:
Done, remove all unused errors from `BaseVerifier`